src/section-function-basics.ptx

```diff
@@ -839,10 +839,9 @@
                                             grid=major,
                                             xtick={-2,...,4},
                                             ytick={5,10,...,15},
-                                            minor ytick={0,1,...,20},
-                                            ]
-                                        \addplot+[dashed,domain=-2.3:4.3] {x^2+1};
-                                        \addplot[soliddot]coordinates{ (-2,5) (-1,2) (0,1) (1,2) (2,5) (3,10) (4,17)};
+                                            minor ytick={0,1,...,20}]
```

Comment:
This can stay as it is, but it's best practice to leave these sorts of option lists with a comma at the end. It just makes it easier to add additional options if the need arises. You just always put the new option with a comma at the end of the line, and never worry about if a comma was on the previous line. Similarly, it's nice to put the closing delimiter on its own line.

Reply:
And by easier, I mean partly when doing it by hand, but definitely when doing anything with regex.